tests/unittests/test_model_zoo.py

```diff
@@ -193,7 +193,15 @@ def test_segmentation_models():
+    _test_model_list(models, ctx, x)
```

Comment:
no, it's testing normal version inference, I mean a test to assert that the weights from normal version can be loaded to sync bn version.
Our test CI currently only use 1 GPU, so there's no way to test inference of sync bn mobilenet. We will add that later.